src/pint/models/timing_model.py

```diff
@@ -228,6 +221,62 @@ def __init__(self, name="", components=[]):
         self.add_param_from_top(
             MJDParameter(name="FINISH", description="End MJD for fitting"), ""
         )
+        self.add_param_from_top(
+            strParameter(
+                name="INFO",
+                description="Tells TEMPO to write some extra information about frontend/backend combinations; -f is recommended",
+            ),
+            "",
+        )
+        self.add_param_from_top(
+            strParameter(
+                name="TIMEEPH",
+                description="Time ephemeris to use for TDB conversion; for PINT, always FB90",
+            ),
+            "",
+        )
+        self.add_param_from_top(
+            strParameter(
+                name="T2CMETHOD",
+                description="Something to do with time conversion; for PINT, always TEMPO",
```

Comment:
"Method for transforming from terrestrial to celestial frame (IAU2000B/TEMPO)."

Reply:
Is PINT really TEMPO? Or is it IAU2000B?

Reply:
Ah, thanks. And I don't know how PINT does it, exactly, but I agree IAU2000B sounds more plausible.